static/app/views/starfish/components/slideOverPanel.tsx

```diff
@@ -10,23 +10,24 @@ import useOrganization from 'sentry/utils/useOrganization';
 const PANEL_WIDTH = '50vw';
 
 type SlideOverPanelProps = {
+  analytics: boolean;
   children: React.ReactNode;
   collapsed: boolean;
 };
 
 export default forwardRef(SlideOverPanel);
 
 function SlideOverPanel(
-  {collapsed, children}: SlideOverPanelProps,
+  {collapsed, children, analytics}: SlideOverPanelProps,
   ref: ForwardedRef<HTMLDivElement>
 ) {
   const {query} = useLocation();
   const organization = useOrganization();
   useEffect(() => {
-    if (!collapsed) {
+    if (!collapsed && analytics) {
       trackAnalytics('starfish.panel.open', {organization});
     }
-  }, [query, collapsed, organization]);
+  }, [query, collapsed, organization, analytics]);
```

Comment:
I think instead of flipping off analytics, we should pass down an `onOpen` function and move the analytics up to wherever this is being called in Starfish. If we don't have our own analytics then we just don't have to implement the `onOpen`